### PR TITLE
Adevolumetype

### PR DESCRIPTION
**Pull request by msmbaldwin**
## Checklist

- [ ] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md)
- [ ] The title of the PR is clear and informative
- [ ] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md#updating-the-change-log)
- [ ] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (Microsoft internal partners only)
    - [ ] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/help-generation.md)

### CODE REVIEW

**Comment by adxsdkps** [comment]
Can one of the admins verify this patch?

File: src/Compute/Compute/help/Set-AzVMDiskEncryptionExtension.md [hunk 1]
```diff
@@ -50,11 +50,12 @@ Set-AzVMDiskEncryptionExtension [-ResourceGroupName] <String> [-VMName] <String>
 The **Set-AzVMDiskEncryptionExtension** cmdlet enables encryption on a running infrastructure as a service (IaaS) virtual machine in Azure.
 This cmdlet enables encryption by installing the disk encryption extension on the virtual machine. 
 
-The *VolumeType* parameter is required when encrypting Linux virtual machines, and must be set to a value ("Os", "Data", or "All") supported by the Linux distribution. The *VolumeType* parameter can be omitted when encrypting Windows virtual machines.
```
Comment (jackrichins) [inline src/Compute/Compute/help/Set-AzVMDiskEncryptionExtension.md]: Isn't volume type still required for Linux VMs? @ejarvi @mayank88mahajan

File: src/Compute/Compute/help/Set-AzVMDiskEncryptionExtension.md [hunk 2]
```diff
@@ -565,10 +566,9 @@ Accept wildcard characters: False
 ```
 
 ### -VolumeType
-Specifies the type of virtual machine volumes on which to perform the encryption operation.
-Allowed values for Windows virtual machines: All, OS, and Data.
-The only allowed value for Linux virtual machines: Data.
-The VolumeType parameter is required when encrypting Linux virtual machines, but can be omitted when encrypting Windows virtual machines.
+Specifies the type of virtual machine volumes on which to perform encryption operation: OS, Data, or All. If this parameter is omitted, the operation defaults to All.
```
Comment (jackrichins) [inline src/Compute/Compute/help/Set-AzVMDiskEncryptionExtension.md]: Default for Linux is Data, isn't it?

File: src/Compute/Compute/help/Set-AzVmssDiskEncryptionExtension.md [hunk 3]
```diff
@@ -274,7 +274,9 @@ Accept wildcard characters: False
 ```
 
 ### -VolumeType
-Type of the volume (OS or Data) on which to perform encryption operation. This parameter is required when encrypting Linux virtual machines, and must be set to a vlue supported by the Linux distribution. This parameter can be omitted when encrypting Windows virtual machines.
+Specifies the type of virtual machine volumes on which to perform encryption operation: OS, Data, or All. If this parameter is omitted, the operation defaults to All.
```
Comment (jackrichins) [inline src/Compute/Compute/help/Set-AzVmssDiskEncryptionExtension.md]: No default for Linux I believe. It must be set to Data explicitly.

File: src/Compute/Compute/help/Set-AzVMDiskEncryptionExtension.md (hunk 2 above)
Comment (ejarvi) [reply src/Compute/Compute/help/Set-AzVMDiskEncryptionExtension.md]: Linux currently defaults to All

File: src/Compute/Compute/help/Set-AzVMDiskEncryptionExtension.md (hunk 1 above)
Comment (ejarvi) [reply src/Compute/Compute/help/Set-AzVMDiskEncryptionExtension.md]: Volume type is still required for Linux

File: src/Compute/Compute/help/Set-AzVMDiskEncryptionExtension.md (hunk 2 above)
Comment (ejarvi) [reply src/Compute/Compute/help/Set-AzVMDiskEncryptionExtension.md]: In the All case, OS + Data will be encrypted if the prerequisite setup steps for data disk encryption have been performed.  If the prerequisites have not been met, data disks, even if present, will not be encrypted.

**Pull request closed without merging**